testsupport/init.go

```diff
@@ -62,11 +67,71 @@ func waitForOperators(t *testing.T) {
 	kubeconfig, err := util.BuildKubernetesRESTConfig(*apiConfig)
 	require.NoError(t, err)
 
+	// creating another config which is used for creating resclient only,
+	//so that the main kubeconfig is not altered
+	restkubeconfig, err := util.BuildKubernetesRESTConfig(*apiConfig)
+	require.NoError(t, err)
+
 	cl, err := client.New(kubeconfig, client.Options{
 		Scheme: schemeWithAllAPIs(t),
 	})
 	require.NoError(t, err)
 
+	// Check if there is already a service account present for e2e test
+	sa := &corev1.ServiceAccount{}
+	err = cl.Get(context.TODO(), types.NamespacedName{Namespace: hostNs, Name: "e2e-test"}, sa)
+
+	// If not found proceed to create the e2e service account and the cluster role binding
+	if errors.IsNotFound(err) {
+		fmt.Printf("No Service Account for e2e test found, proceeding to create it")
```

Comment:
Let's use `t.Log` for printing some information in the tests:
```suggestion
		t.Log("No Service Account for e2e test found, proceeding to create it")
```

Reply:
done